src/popup/router/pages/TransactionDetails.vue

```diff
@@ -229,6 +229,17 @@ export default {
       .span-2-columns {
         grid-column-end: span 2;
       }
+
+      .tip-url {
+        width: 100%;
+
+        .link-button {
+          display: block;
+          white-space: nowrap;
+          overflow: hidden;
+          text-overflow: ellipsis;
```

Comment:
```suggestion
```
I suggest to review #1264 and remove this lines because we might want to have same behavior in other places.